package.json

```diff
@@ -159,7 +159,7 @@
     "react-native-tab-view": "^3.5.2",
     "react-native-url-polyfill": "^2.0.0",
     "react-native-view-shot": "3.8.0",
-    "react-native-vision-camera": "^2.16.2",
+    "react-native-vision-camera": "2.16.5",
```

Comment:
what's this change for?

Reply:
It was required, because of some C++ errors during build on android in HybridApp

Reply:
Side note: this will be updated to v3 in https://github.com/Expensify/App/pull/28914

Reply:
This also updates iOS pod. Please commit Podfile.lock

<img width="1105" alt="Screenshot 2024-02-02 at 10 29 25 PM" src="https://github.com/Expensify/App/assets/108292595/8fc4526f-6120-4393-be26-79b9f3280e5a">

Reply:
This version bump caused deploy blocker - https://github.com/Expensify/App/issues/36098.
@staszekscp can you please check urgently?

Reply:
> some C++ errors during build on android in HybridApp

Which version fixed this? among 2.16.3, 2.16.4, 2.16.5

And from which version #36098 started happening?